renderer/reducers/payment.js

```diff
@@ -16,6 +16,7 @@ export const RECEIVE_PAYMENTS = 'RECEIVE_PAYMENTS'
 export const SEND_PAYMENT = 'SEND_PAYMENT'
 export const PAYMENT_SUCCESSFUL = 'PAYMENT_SUCCESSFUL'
 export const PAYMENT_FAILED = 'PAYMENT_FAILED'
+export const DEC_PAYMENT_RETRIES = 'DEC_PAYMENT_RETRIES'
```

Comment:
Lets rename to `DECREMENT_PAYMENT_RETRIES` for clarity (same for other instances of `dec`)